openapi/definitions.yaml

```diff
@@ -322,14 +322,42 @@ BlockedInboxOrChannels:
   description: |-
     All the notification channels blocked by the user.
     Each channel is related to a specific service (sender).
+NewProfile:
+  type: object
+  description: |-
+    Describes a new citizen's profile.
+    Used for profile creation.
+  properties:
+    email:
+      $ref: "#/EmailAddress"
+    is_email_validated:
+      $ref: "#/IsEmailValidated"
+  required:
+    - is_email_validated
 Profile:
+  type: object
   description: |-
-    Describes the citizen's profile, mostly interesting for preferences
-    attributes.
-  x-one-of: true
-  allOf:
-    - $ref: "#/LimitedProfile"
-    - $ref: "#/ExtendedProfile"
```

Comment:
is `Profile` the same as `ExtendedProfile` ?

Reply:
Read above